source/_integrations/matter.markdown

```diff
@@ -107,18 +104,29 @@ Note: The section below mentions third-party Thread border routers such as the N
     - If you are already running the Matter server in another add-on, in or a custom container:
       - Deselect the checkbox, then select **Submit**.
       - In the next step, provide the URL to your Matter server.
+
+#### Check logos on the device
+
+- On the device packaging, check for both the Matter logo and for either the Wi-Fi or the {% term Thread %} logo.
+- Check if the QR code is only on the packaging or if it is also on the device.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store the image and the numerical code in a save place.
+  - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
+
+#### Prepare Android or iPhone
+
 - Have either an Android or iPhone ready and Bluetooth enabled. For information why Bluetooth is required, refer to the section on [Bluetooth used during commissioning](#bluetooth-used-during-commissioning):
   - Android:
     - Have an Android phone (a full Android, not F-Droid).
     - Have the latest version of the Home Assistant Companion app installed.
-    - Have Google Home app installed on the Android.
+    - Have the Google Home app installed on the Android.
     - We are not going to add the new device to Google Home. The app is needed because Google included the Matter SDK there.
```

Comment:
I've double checked wiith @jpelgrom, this really should not be necessary.

There were reports from users which had problems in the past, but reinstalling the HA App should help too. Also that was early days, I doubt a bit that this is still a problem.

Suggestion: Let's see if this problem appears again now that Matter support on Android matured. If so, we can add a hint to Troubleshooting.

But I'd suggest to drop this prerequisites here.

```suggestion
```

/cc @marcelveldt